gigadb/app/tools/gitlab-runner/scripts/delete_runner_cache.sh

```
@@ -0,0 +1,46 @@
+
+# Log disk usage before cleanup
+log_message "Disk usage before cleanup:"
+du -h "$CACHE_DIR" 2>&1 | tee -a "$LOG_FILE"
+
+# Count files/directories to be deleted
+ITEM_COUNT=$(find "$CACHE_DIR" -mindepth 1 | wc -l)
+log_message "Items to be deleted: $ITEM_COUNT"
+
+# Perform cleanup
+log_message "Starting deletion process..."
+if rm -rf "$CACHE_DIR"/* 2>&1 | tee -a "$LOG_FILE"; then
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)